chrooted-iso.sh

```diff
@@ -56,4 +52,8 @@ echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
 echo "LANG=en_US.UTF-8" > /etc/locale.conf
 locale-gen
 
+# Onyx as default gdm session
```

Comment:
As long as this is tested im fine with all these changes

Reply:
@axtloss ?

Reply:
Yeah this is tested, I thought you were saying something about the onyx gdm thing